src/graphql/public/schema.graphql

```diff
@@ -79,6 +80,16 @@ type AccountUpdateDisplayCurrencyPayload {
   errors: [Error!]!
 }
 
+input AccountUpdatePushNotificationSettingsInput {
+  disabledPushNotificationTypes: [PushNotificationType]!
+  pushNotificationsEnabled: Boolean!
+}
+
+type AccountUpdatePushNotificationSettingsPayload {
+  account: ConsumerAccount
```

Comment:
do we not use `Account` instead for the other similar queries?

Reply:
At the moment we use `ConsumerAccount` for the other similar queries but maybe this is by mistake?